.eslintrc

```diff
@@ -1,3 +1,5 @@
+extends: eslint:recommended
```

Comment:
Excellent question...

| Rule | Value | [`eslint:recommended`](https://github.com/eslint/eslint/blob/master/conf/eslint.json) | Notes
|----|:----:|:----:|----|
| `curly` | 0 | "off" | Same.
| `eqeqeq` | 2 | "off" | -
| `guard-for-in` | 2 | "off" | -
| `indent` | [2, 4, {SwitchCase: 1}] | "off" | -
| `no-bitwise` | 2 | "off" | -
| `no-caller` | 2 | "off" | -
| `no-empty` | 2 | "error" | Same.
| `no-extend-native` | 2 | "off" | -
| `no-irregular-whitespace` | 2 | "error" | Same.
| `no-undef` | 2 | "error" | Same.
| `no-unused-vars` | 2 | "error" | Same.
| `quotes` | [2, single] | "off" | -
| `strict` | [2, function] | "off" | -

Looks like we have 5 rules that are currently the default in eslint:recommended. We can remove those from our `rules` in .eslintrc, or leave them if we are concerned that they ever may change in future versions of the recommended rules.